flow/ProtocolVersion.h

```diff
@@ -31,14 +31,14 @@
 // used and should not be changed from 0.
 //                                                         xyzdev
 //                                                         vvvv
-constexpr uint64_t currentProtocolVersionValue = 0x0FDB00B071010000LL;
+constexpr uint64_t currentProtocolVersionValue = 0x0FDB00B072010000LL;
```

Comment:
This isn't required, but the dev version can be reset to 0 when increasing higher bytes in the protocol version.

```suggestion
constexpr uint64_t currentProtocolVersionValue = 0x0FDB00B072000000LL;
```

Reply:
For what it's worth, I personally would like for this to be reset back to 0. I suspect the reason it has been at 1 for several versions is that it got incremented and then we neglected to reset it.